RecoTracker/IterativeTracking/python/PixelPairStep_cff.py

```diff
@@ -333,30 +336,19 @@
 pixelPairStep.mva.GBRForestLabel = 'MVASelectorIter2_13TeV'
 pixelPairStep.qualityCuts = [-0.2,0.0,0.3]
 
-#LWTNN selector
+trackingPhase1.toModify(pixelPairStep, mva=dict(GBRForestLabel = 'MVASelectorPixelPairStep_Phase1'))
+
 from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
 from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
-trackingPhase1.toReplaceWith(pixelPairStep, TrackLwtnnClassifier.clone(
-    src='pixelPairStepTracks',
-    qualityCuts=[-0.6, -0.1, 0.4],
-))
-
-highBetaStar_2018.toReplaceWith(pixelPairStep, TrackMVAClassifierPrompt.clone(
+trackdnn.toReplaceWith(pixelPairStep, TrackLwtnnClassifier.clone(
     src='pixelPairStepTracks',
-    qualityCuts = [-0.95,0.0,0.3],
-    mva=dict(GBRForestLabel = 'MVASelectorPixelPairStep_Phase1')
-))
-
-pp_on_AA_2018.toReplaceWith(pixelPairStep, TrackMVAClassifierPrompt.clone( 
-    src='pixelPairStepTracks',
-    qualityCuts = [-0.2, 0.0, 0.98],
-    mva=dict(GBRForestLabel = 'MVASelectorPixelPairStep_Phase1') 
+    qualityCuts=[-0.6, -0.1, 0.4]
 ))
 
+highBetaStar_2018.toModify(pixelPairStep,qualityCuts = [-0.95,0.0,0.3])
+pp_on_AA_2018.toModify(pixelPairStep, qualityCuts = [-0.2, 0.0, 0.9])
```

Comment:
the tight value was recently changed to 0.98 in #26217. Please recover the correct value.

Reply:
Fixed to the correct value